packages/embedded/src/router.ts

```diff
@@ -16,7 +16,13 @@ export function createRouter<E extends Env>(
   // We therefore remove the apiKey
   const { apiKey, fpxEndpoint, ...sanitizedOptions } = options;
 
-  app.route("/api", createApiRoutes(apiKey, fpxEndpoint));
+  if (apiKey) {
+    app.route("/api", createApiRoutes(apiKey, fpxEndpoint));
+  } else {
+    app.use("/api/*", async (c) => {
+      return c.json({ error: "Fiberplane API key is not set" }, 402);
```

Comment:
nit but shouldn't this be 401 instead?

Reply:
explained in slack the reasoning for 402, but i can easily change it